pkg/cmd/start/root.go

```diff
@@ -57,17 +57,22 @@ var startCmd = &cobra.Command{
 		config, err := project.ConfigFromProjectPath("")
 		utils.CheckErr(err)
 
-		proj, err := project.FromConfig(config)
-		utils.CheckErr(err)
+		history := &history.History{
+			ProjectDir: config.Dir,
+		}
+
+		proj := &project.Project{
+			Name:    config.Name,
+			Dir:     config.Dir,
+			History: history,
+		}
```

Comment:
Does History need to be apart of project? Looking at the references all access to History are as a public field on Project.

Reply:
yes otherwise we get a nil pointer error elsewhere. Could probably be fixed by making more methods detached, this was less churn in the PR changes before our refactor tasks

Reply:
Discussed, this will be considered in the refactor